package.json

```diff
@@ -6,7 +6,8 @@
     "check:component": "run-s \"php-clean:{1}\" \"render:* {1} {2}\" \"php-prettify:{1}\" \"diff {1} {2}\" --",
     "clean": "run-p twing:clean php:clean",
     "diff": "node_modules/@ecl-twig/php-storybook/scripts/diff.js",
-    "diff:ecl": "node_modules/@ecl-twig/php-storybook/scripts/ecl-diff.js --interactive",
+    "diff:ecl": "node_modules/@ecl-twig/ecl-diff/ecl-diff.js --interactive",
```

Comment:
don't think @kalinchernev that i haven't tried, actually that was changed in the past and then reverted it, i think, i will try again to run these scripts with lerna, but it was failing, that's why you see this "iconsistency"

Reply:
So, basically....i went through this again, i managed to use the diff script with lerna but with all the others i had issues and could not fix them:
- ecl-diff is an interactive script, i've finally found a way to get some output with lerna run --stream but 
<details>
lerna --scope "@ecl-twig/ecl-diff" run ecl-diff --stream -- --interactive
lerna notice cli v3.22.1
lerna notice filter including "@ecl-twig/ecl-diff"
@ecl-twig/ecl-diff: $ ./ecl-diff.js --interactive
@ecl-twig/ecl-diff: ? The component you want to diff (Use arrow keys)
@ecl-twig/ecl-diff: ❯ accordion 
@ecl-twig/ecl-diff:   accordion2 
@ecl-twig/ecl-diff:   blockquote 
@ecl-twig/ecl-diff:   breadcrumb 
@ecl-twig/ecl-diff:   breadcrumb-core 
@ecl-twig/ecl-diff:   breadcrumb-harmonised 
@ecl-twig/ecl-diff:   breadcrumb-standardised 
@ecl-twig/ecl-diff: error Command failed with exit code 130.
</details>
- ecl-diff-full, well, we would not get any output in the console at the moment.
- php:copy which is a script running ncp doesn't copy the file when i run it with lerna run, although it ends succesfully
- php:dist-fix which is a script running replaceInFile doesn't write the files when i run it with lerna although it endssuccesfully

in the package.json of php-storybook you find indeed already all the scripts defined because i've tried in the past, then i saw that the php deplyoment was failing and reverted the two call to the related scripts.